tests/unit-tests/internal/emails/test-class-email-settings-tab.php

```diff
@@ -43,4 +54,64 @@ public function testTabContent_WhenCalledWithAnotherTab_ReturnsDefaultContent()
 		/* Assert. */
 		self::assertSame( 'a', $content );
 	}
+
+	public function testTabContent_WhenInStudentSubtabAndHasAnEmailOfThatGroup_ReturnsContantWithTheEmail() {
```

Comment:
*ReturnsContent

Reply:
All the typos were fixed in 0822c191f0901e4c83a5ef88e7c5779ba6b7428e. Thank you!